lib/execution.ex

```diff
@@ -60,16 +57,20 @@ defmodule Firefighter.Execution do
     start(ids) |> push(id)
   end
 
-  defp to_record(%__MODULE__{event_uuid: event_uuid, elapsed: elapsed, data: data}) do
-    record =
-      %{event_uuid: event_uuid, elapsed: elapsed}
-      |> Map.merge(data)
-
-    json().encode!(record)
+  defp to_record(
+         %__MODULE__{event_uuid: event_uuid, event_time: event_time, data: data} = execution
+       ) do
+    %{
+      event_uuid: event_uuid,
+      event_time: event_time |> DateTime.to_iso8601(),
+      elapsed: DateTime.diff(current_time(), event_time)
```

Comment:
Does `DateTime.diff/2` return an integer for the difference?

Reply:
yes (according to the docs, already in seconds, which is what was being hand-calculated in the previous version).